preprocessing/text_data/04_preprocessing_text_data_v3.ipynb

```diff
@@ -0,0 +1,1955 @@
+{
```

Comment:
<blockquote>Let's check for most rare words now</blockquote><p>Should be</p><blockquote>Let's check for the most rare words now</blockquote><p> </p><p>In this sentence, "it" is vague:</p><blockquote>It is mostly wrong spelling words, which we will later auto correct, but we can add them to our stop words list as well.</blockquote><p>Suggest updating to something like for style and readability: </p><blockquote>In this example, infrequently used words mostly consist of misspelled words, which we will later auto correct, but we can add them to our stop words list as well.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>